Jenkinsfile.conda

```diff
@@ -1,2 +1,2 @@
 @Library('JenkinsShared')_
-CondaPipeline(["ts_config_atcalsys"], "ts-atwhitelightsource", "lsst.ts.ATWhiteLightSource")
\ No newline at end of file
+CondaPipeline(["ts_config_atcalsys"], "ts-atwhitelight", "lsst.ts.atwhitelight")
```

Comment:
You can easily rename the repository. I agree with removing the "Source" suffix, but renaming the repository (somewhere under settings in GitHub) will make it more consistent (I renamed recently ts_AirCompressors to ts_MTAirCompressor and it was a smooth ride).

Reply:
Yes. Right after merging.